changelog/27578.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:change
+cli: The `-dev-three-node` and `-dev-four-cluster` CLI options have been removed.
```

Comment:
```suggestion
cli: The undocumented `-dev-three-node` and `-dev-four-cluster` CLI options have been removed.
```

Doubt it matters, but we never documented this as a supported thing for users to do, did we?

Reply:
We never did, yeah, you're right that this would be a good addition